README.rst

```diff
@@ -1,6 +1,15 @@
 ======
 segtok
 ======
+
+.. image:: https://img.shields.io/pypi/v/segtok.svg
+    :target: https://pypi.python.org/pypi/segtok
+
+.. image:: https://img.shields.io/pypi/l/segtok.svg
+
+.. image:: https://img.shields.io/travis/kmike/segtok.svg?branch=py2
```

Comment:
To enable a badge for your repo first enable Travis CI (login to Travis with your github account, go to Accounts and turn CI on for segtok), then replace this line with `https://img.shields.io/travis/fnl/segtok.svg?branch=master`.

To check how it looks see https://github.com/kmike/segtok/tree/py2.